test/e2e-layering/onclusterbuild_test.go

```diff
@@ -225,3 +228,91 @@ func prepareForTest(t *testing.T, cs *framework.ClientSet, testOpts onClusterBui
 	_, err = helpers.WaitForRenderedConfig(t, cs, testOpts.poolName, "00-worker")
 	require.NoError(t, err)
 }
+
+func TestSSHKeyAndPasswordForOSBuilder(t *testing.T) {
+	cs := framework.NewClientSet("")
+
+	// label random node from pool, get the node
+	unlabelFunc := helpers.LabelRandomNodeFromPool(t, cs, "worker", "node-role.kubernetes.io/layered")
+	osNode := helpers.GetSingleNodeByRole(t, cs, layeredMCPName)
+
+	// prepare for on cluster build test
+	prepareForTest(t, cs, onClusterBuildTestOpts{
+		imageBuilderType: build.OpenshiftImageBuilder,
+		poolName:         layeredMCPName,
+		customDockerfiles: map[string]string{
```

Comment:
suggestion: Since the rest of this particular test does not look for the custom Dockerfile content, for simplicity sake, make this:

```golang
customDockerfiles: map[string]string{},
```